pkg/cmd/util/tokencmd/request_token.go

```diff
@@ -45,17 +45,14 @@ func RequestToken(clientCfg *kclient.Config, reader io.Reader, defaultUsername s
 
 	osClient.Client = &challengingClient{httpClient, reader, defaultUsername, defaultPassword}
 
-	result := osClient.Get().AbsPath("oauth", "authorize").Param("response_type", "token").Param("client_id", "openshift-challenging-client").Do()
+	result := osClient.Get().AbsPath(server.OpenShiftOAuthAPIPrefix, osinserver.AuthorizePath).Param("response_type", "token").Param("client_id", "openshift-challenging-client").Do()
```

Comment:
It's not this line, but the mutation of the clientCfg in this method is bad.  Make a copy, avoid the mutation and leave it to the caller to decide what to do with the bearer token.  If this ripples too far, add a TODO on RequestToken